src/core/symbol/SymbolMessageUtils.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * The Message type. Supported supply types are:
+ * -1: RawMessage (no type appended)
+ * 0: PlainMessage
+ * 1: EncryptedMessage.
+ * 254: Persistent harvesting delegation.
+ */
+
+import { Converter } from '@utils';
+
+export enum MessageType {
+    PlainMessage = 0x00,
+    EncryptedMessage = 0x01,
+    PersistentHarvestingDelegationMessage = 0xfe,
+}
+
+// TODO add more methods to serialize and deserialize messages for each Message type.
+export class SymbolMessageUtils {
+    public static plain(plainText: string): Uint8Array {
+        return Converter.concat(Uint8Array.of(MessageType.PlainMessage), Converter.utf8ToUint8(plainText));
+    }
+
+    // public static encrypted(keyPair:KeyPair, plainText: string): Uint8Array {
```

Comment:
not being used?

Reply:
Not yet, it will be part of the Message API eventually. Do you want to handle the Message Api in this pr?